go.mod

```diff
@@ -49,3 +51,7 @@ require (
 	gopkg.in/macaroon.v2 v2.0.0
 	gopkg.in/mgo.v2 v2.0.0-20180705113604-9856a29383ce // indirect
 )
+
+replace github.com/lightningnetwork/lnd/ticker v1.0.0 => ./ticker
```

Comment:
is this a standard way of using submodules?

Reply:
I'll use an example to hopefully get the point across better:

Let's say we wish to rename the `Ticker` interface to something else. Without this replace directive, we would have to commit the changes to the `ticker` package, tag them, and update the version used within the project's root `go.mod` in order to use those changes elsewhere in the project. With the directive in place, we can avoid doing so.

I guess it's all a matter of preference. We can remove it and everything will still work fine, the extra hurdle will exist if  we decide to no include the replace directive.

Reply:
Because the root project still depends on them. Also, a replace directive for a package can't exist without a require directive for the same package.